Cargo.toml

```diff
@@ -6,7 +6,7 @@ license = 'Apache-2.0'
 name = "hydra-dx-math"
 description = "A collection of utilities to make performing liquidity pool calculations more convenient."
 repository = 'https://github.com/galacticcouncil/hydradx-math'
-version = "4.9.1"
+version = "5.0.0"
```

Comment:
it looks like the interface has not changed. so i would not bump the major version